testing/scan_by_key.cu

```diff
@@ -374,15 +374,17 @@ void TestInclusiveScanByKey(const size_t n)
     thrust::host_vector<int> h_keys(n);
```

Comment:
Can you delete the `#if 0`'d "nvbug 1541533 WAR" that is right above this line?

Reply:
Done.